### ISSUE TITLE
Extend estimators.md to cover Keras

### ISSUE THREAD

**Issue by bhack** [tensorflow/tensorflow#14879]
Can you extend the estimators.md tutorial to cover `tf.keras.estimator.model_to_estimator` use case?

**Comment by bhack** [comment]
I think that could be useful to have a full example with estimator, tf.keras and dataset API now that all these API are in core. See also https://github.com/tensorflow/tensorflow/issues/8787#issuecomment-337297281 /cc @fchollet

**Comment by bhack** [comment]
A nice example to add to documentation is [something like this one](https://www.dlology.com/blog/an-easy-guide-to-build-new-tensorflow-datasets-and-estimator-with-keras-model/) but including also the Dataset API.

**Comment by tatatodd** [comment]
Seems reasonable to me, can you comment on this @fchollet?  Thanks!

**Comment by fchollet** [comment]
Yes, I agree that covering the use case "how can I define a custom Estimator using the Keras API" would be great to have in the main tutorial. How should we proceed?

**Comment by bhack** [comment]
I think that @Tony607's Notebook I linked it is quite complete. But I don't know if he want to contribuite it.
Probably also a 2nd flavour with custom loss, explicit model_,fn, Tensorboard summaries could be interesting to have more control for power users that still want to rely on the high level API. There was  https://github.com/mjlaali/keras-estimator/ but still on the contrib interface.

**Comment by bhack** [comment]
@fchollet Also what it s needed to document where the additonal summaries for Tebsorboard need to be inserted. Cause with this API call we don't have an explicit model_fn to write, so what is the best practice for placing addtional summaries?

**Comment by bhack** [comment]
Also just to point other missing documentation we can specify how to mix [tf metrics in tf.keras](https://stackoverflow.com/questions/45947351/how-to-use-tensorflow-metrics-in-keras) cause was requested in some Keras ticket but still not officially documented. We have some TF metrics that are not directly covered by tf.keras metrics counterpart so could be useful to document its use now the we are in the same tf root namespace.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
A member of the TensorFlow organization has replied after the stat:awaiting tensorflower label was applied.
